src/app/components/navigation/navigation.component.html

```diff
@@ -50,15 +50,18 @@
            routerLinkActive="routerLinkActive">
           <span class="pl-3">Algorithm Relation Types</span>
         </a>
-        <a class="sub-tab" mat-list-item [routerLink]="'./pattern-relation-types'"
+        <a *ngIf="patternAtlasFeatureIsSet" class="sub-tab" mat-list-item [routerLink]="'./pattern-relation-types'"
```

Comment:
Okay, I guess we could enhance this here, if we use the whole config element from the service here. If you make the service public, you could enable the change listener to detect, that the config has changed. Try to use `configData.features.patternAtlas` instead of `patternAtlasFeatureIsSet`.

Reply:
Thanks for this valuable hint!